### DISCUSSION TITLE
[✨] action$ actionPath url params behave differently for <Form> vs <form>

### DISCUSSION THREAD

**Discussion by tuurbo** [QwikDev/qwik-evolution#170] [Proposals For Qwik]
### Is your feature request related to a problem?

When submitting a form with `<Form action={useAction} />` the actionPath keeps the current urls search params. If you submit a form with `<form action={useAction.actionPath} />` you lose the urls search params.

To keep url search params, i currently have to append to the `formAction.actionPath`
```tsx
export const useAction = action$((formData, { url }) => {
  return {
    success: true,
  };
});

<form action={formAction.actionPath + '&foo=111'} method="post">
  <button>submit</button>
</form>
```

### Describe the solution you'd like

Should both ways of submitting a form behave the same? If so, maybe allow passing an option to disable keeping params (not sure if there is a use case for this)?

```tsx
const formAction = useAction({
  preserveUrlSearchParams: false,
});
```

### Describe alternatives you've considered

Also, for a separate issue, maybe there is a use case for an option like this?
```tsx
const formAction = useAction({
  dangerouslyAllowCrossSite: true,
});
```

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
